src/gravel/controllers/nodes/systemdisk.py

```diff
@@ -102,17 +136,34 @@ def mounted(self):
                 return True
         return False
 
-    async def lvm(self, args: str) -> None:
-        cmd: List[str] = ["lvm"] + shlex.split(args)
-        retcode, _, err = await aqr_run_cmd(cmd)
+    async def exists(self) -> bool:
+        """Checks whether a System Disk exists in the system."""
+
+        lvmcmd = "lvs --noheadings -o vg_name,lv_name @aquarium"
+        try:
+            out = await lvm(lvmcmd)
```

Comment:
I think it should be something like that:
```suggestion
            out = await lvm(["lvs", "--noheadings", "-o", "vg_name,lv_name", "@aquarium"]])
```

Reply:
I'd also prefer using an argv rather than shelx ..

Reply:
While I understand the point of getting a list of strings, why would manually passing the list of strings be better than let the shell lexicon parser, or more reliable?